src/core/installAddon.js

```diff
@@ -122,11 +126,23 @@ export function makeProgressHandler(dispatch, guid) {
         type: INSTALL_CANCELLED,
         payload: { guid },
       });
+
+      _tracking.sendEvent({
+        action: getAddonTypeForTracking(type),
+        category: getAddonEventCategory(type, INSTALL_CANCELLED_ACTION),
```

Comment:
is it possible to track this for open search and lang?

Reply:
also, i can't seem to comment above (around like 119) - should this sendEvent INSTALL_ERROR_ACTION

Reply:
oh I guess this is below?

Reply:
no I don't think it is possible because we do not have such an event.

Reply:
You mean sending an event for `DOWNLOAD_FAILED`?

Reply:
right or maybe that is covered below.

I haven't been able to test the error stuff myself yet..

Reply:
> is it possible to track this for open search and lang?

I saw when i tried install open search, there was a cancel option (altho it doesn't look a little different from a regular extension).

I guess you can disregard my comment above INSTALL_ERROR_ACTION for now as you said you mentioned that would be revisiting this :) Basically tho I wasn't sure if we should be tracking this altho I realized you might have already set that up somewhere else (in install catch?)

Reply:
unfortunately, we cannot do anything for open search add-ons. I tried and failed. The `AddSearchProvider` opens a sort of confirm popup but there is no return value available and it is not a promise-based API.